source/MRMesh/MRSphereObject.h

```diff
@@ -9,7 +10,7 @@ namespace MR
 
 /// Object to show sphere feature, position and radius are controlled by xf
 /// \ingroup FeaturesGroup
-class MRMESH_CLASS SphereObject : public FeatureObject
+class MRMESH_CLASS SphereObject : public ObjectWithMeasurements<FeatureObject, RadiusVisualizePropertyType(1)>
```

Comment:
`RadiusVisualizePropertyType(1)` what is `(1)`

Reply:
How many measurements of this kind.